packages/nextjs/test/config/webpack/webpackPluginOptions.test.ts

```diff
@@ -52,6 +52,10 @@ describe('getWebpackPluginOptions()', () => {
           env: 'my-env',
         },
       },
+      bundleSizeOptimizations: {
+        excludeTracing: true,
+        excludeReplayShadowDom: false,
+      },
```

Comment:
l: Any reason we added this here? I don't see any changes to assertions.

Reply:
Yeah, makes sense to separate this. I created another test just for those options.